SDKTest/SessionTests.swift

```diff
@@ -22,7 +22,7 @@ class SessionTests: XCTestCase {
 
     func testConfigureEndpoint() throws {
         #if DEBUG
-        XCTAssertTrue(NeuroID.getCollectionEndpointURL() == "https://receiver.neuroid.cloud/c")
```

Comment:
Does this method configure the dev endpoints before running the above test (NeuroIDClassTests)?